operator.js

```diff
@@ -0,0 +1,37 @@
+// 04.08.2020
+// first part
+
+
+let snackServings = 10;
+let guests = 20;
+
+console.log(snackServings + guests);
+console.log(guests - snackServings);
+console.log(snackServings - guests);
+console.log(snackServings * guests);
+
+let drinks = 10;
+console.log(snackServings * guests) / drinks;
+let resultOne = snackServings * guests / drinks;
+console.log(resultOne);
+
+let monkeys = 15;
+let bananas = 9;
+console.log(monkeys / bananas);
+
+let monkeyfights = 20;
+console.log(monkeys + bananas) * monkeyfights;
+let resultTwo = monkeys + bananas * monkeyfights;
+console.log(resultTwo);
+
+// Increment to Monkeys
+monkey = 20;
+
+// Decrement to Banana
+bananas =5 ;
+
+
+console.log(monkeys - bananas);
+let happiness = monkeys - bananas;
+console.log(happiness + monkeyfights);
+console.log(resultOne / resultTwo);
```

Comment:
//11. Print the remainder when resultOne is divided by resultTwo to the console.

may we can think to use  %

Reply:
check this out 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Remainder